pt.html

```diff
@@ -201,96 +207,97 @@
 <panel w=500 h=450 fadeInOn="flip_spaced_rep">
 <pic src="pics/intro0.png" sx=950 sy=0></pic>
 <words x=10 y=10 w=430 h=60>
-	And it's <i>awesome</i>.
-	I started using Spaced Repetition earlier this year to learn French.
+	E é <i>incrível</i>.
+	Comecei a usar o Repetição Espaçada no início deste ano para aprender francês.
+
 </words>
 <words x=30 y=350 w=430 h=60>
-	In two <i>months</i>, I learnt more words than I did in two <i>years</i>
-	of high school French classes.
+	Em dois <i>meses</i> eu aprendi mais palavras do que em dois <i>anos</i>
+	de aulas de francês no ensino médio.
 </words>
 </panel>
 
 <panel w=500 h=400>
 <pic src="pics/intro0.png" sx=950 sy=450></pic>
 <words x=10 y=10 w=400 h=60>
-	Since then, I've used Spaced Repetition to remember all sorts of things...
+	Desde então, passei a usar a Repetição Espaçada para lembrar de todo tipo de coisa...
 </words>
 <words x=81 y=113 w=100 style="text-align:left" no-bg class="comic_text">
-	ukulele chords
+	acordes de ukulele
 </words>
 <words x=212 y=114 w=100 style="text-align:left" no-bg class="comic_text">
-	computer code
+	código de programação
 </words>
 <words x=380 y=137 w=100 style="text-align:left" no-bg class="comic_text">
-	friends' birthdays
+	aniversário de amigos
 </words>
 <words x=188 y=204 w=270 style="text-align:left" no-bg class="comic_text">
-	anything interesting i find in books, talks, articles, etc!
+	qualquer coisa interessante que eu encontre em livros, palestras, artigos, etc!
 </words>
 <words x=60 y=300 w=400 h=60>
-	...and this lil' memory card game became a core part of my <i>life</i>.
+	...e este pequeno jogo de cartas de memória tornou-se uma parte essencial da minha <i>vida</i>.
 </words>
 </panel>
 
 <panel w=550 h=350>
 <pic src="pics/intro0.png" sx=0 sy=850></pic>
 <words x=10 y=10 w=500 h=30>
-	In short, Spaced Repetition = testing + time.
+	Resumidamente, Repetição Espaçada = ensaio + tempo
 </words>
 <words x=10 y=220 w=500 h=90>
-	You test yourself on a fact repeatedly, spacing out your repetitions over time.
-	(But won't this take forever? Ah, as we'll see later, there's a trick...)
+	Você se testa em um fato repetidamente, espaçando suas repetições ao longo do tempo.
```

Comment:
Não encaixa no balão.

Talvez:
```suggestion
	Você se testa em um fato repetidamente, espaçando ao longo do tempo.
```